docs/sphinx/user_guide/feature/policies.rst

```diff
@@ -531,11 +531,13 @@ explanation along with examples of how they are used can be found in
 
   * ``statement::Lambda< LambdaId, Args...>`` extension of the lambda statement; enabling lambda arguments to be specified at compile time.
 
-  * ``statement::Segs<...>`` argument to a Lambda statement; used to specify which segments in a tuple will be used as lambda arguments.
+  * ``Segs<...>`` argument to a Lambda statement; used to specify which segments in a tuple will be used as lambda arguments.
```

Comment:
@ajkunen I think it would be better to break this list into two lists, one with things in the 'RAJA::statement' namespace and one with things in the 'RAJA' namespace. Also, make it completely obvious in a statement preceding each list which namespace the following items live in.

Reply:
ok, i'll work on that

Reply:
Thanks, it shouldn't take much work.

Reply:
done

Reply:
Thanks. Looks good.